libgstc/python/setup.py

```diff
@@ -44,10 +44,10 @@
                            'tests']),
     scripts=[],
     classifiers=['Development Status :: 3 - Alpha',
-                 'Programming Language :: Python :: 3.5',
-                 'Programming Language :: Python :: 3.6',
-                 'Programming Language :: Python :: 3.7'],
-    python_requires='>=3.5',
+                 'Programming Language :: Python :: 3.7',
+                 'Programming Language :: Python :: 3.8',
+                 'Programming Language :: Python :: 3.9'],
+    python_requires='>=3.7',
```

Comment:
Does `asyncio` not support `3.5`, otherwise we might be breaking support

Reply:
Well basic asyncio does support `3.5`, however [specific features](https://github.com/RidgeRun/gstd-1.x/pull/252#discussion_r721497115) I used have a minimum version of `3.7`.

Reply:
FYI python `3.5` is already end of life, and `3.6` will be end of life [within a couple of months](https://www.python.org/dev/peps/pep-0494/#lifespan) looks like.

Reply:
python 3.6 is [now end of life](https://www.python.org/dev/peps/pep-0494/#lifespan) so requiring python 3.7 shouldn't be an issue anymore right?